conda/recipes/mantid-developer/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set version = load_setup_py_data(setup_file='../../../setup.py', from_recipe_dir=True).get('version', '0.0.0') %}
+
+package:
+  name: mantid-developer
+  version: {{ version }}
+
+requirements:
+  run:
+    - ccache
+    - cmake {{ cmake }}
+    - conda-wrappers # [win]
+    - doxygen {{ doxygen }}
+    - eigen {{ eigen }}
+    - euphonic {{ euphonic }}
+    - graphviz {{ graphviz }}
+    - gsl {{ gsl }}
+    - h5py
+    - hdf5 {{ hdf5 }}
+    - jemalloc {{ jemalloc }} # [linux]
+    - jsoncpp {{ jsoncpp }}
+    - lib3mf # [win]
+    - libboost-devel {{ libboost_devel }}
+    - libboost-python-devel {{ libboost_python_devel }}
+    - libopenblas {{ libopenblas }} # [osx or linux]
+    - librdkafka {{ librdkafka }}
+    - matplotlib {{ matplotlib }}
+    - muparser {{ muparser }}
+    - nexus {{ nexus }}
+    - ninja {{ ninja }}
+    - numpy>=1.24,<1.27 # This is intentionally different to conda_build_config.yaml, will address later.
```

Comment:
Out of interest, why is this different?

Reply:
From Tom:
- We allow numpy package 1.24,1.25,1.26 in our developer environments
- We build all our packages against 1.24